client/src/components/MessageTwo.js

```diff
@@ -0,0 +1,25 @@
+import styled from "@emotion/styled";
+import colors from "../utils/colors";
+import React from "react";
+
+export default function MessageTwo() {
+  return (
+    <Message>
+      Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+      eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+      voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
+      clita kasd gubergren!
+    </Message>
+  );
+}
+
+const Message = styled.div`
+  background: linear-gradient(50deg, ${colors.gradientOne}, #364cf7);
```

Comment:
Maybe the color `#364cf7` can also be saved in the colors object and accessed the same as `gradientOne`. This way, all of the colors in this line are set in the same way.

Reply:
I only need this color in this case, so I decided to use it in this way. 
But for the sake of consistency it would be better to save the color in the colors file as well.